src/bindings/cpp/tests/testcpp_key.cpp

```diff
@@ -13,6 +13,52 @@
 #include <string>
 #include <vector>
 
+/* check if exceptions are thrown if the underlying c-functions return error codes (-1 or NULL) */
+TEST (key, cErrors)
+{
+	Key k ("user:/key", KEY_VALUE, "testkey", KEY_END);
+	Key x = nullptr;
+
+	EXPECT_THROW (x.addName (""), KeyInvalidName);
```

Comment:
The test is confusing. Please separate the tests x==nullptr and addName with invalid name.

Reply:
Please see my response to the previous similar comment on confusing tests, thank you!